lib/DDGC/DB/Result/User/Blog.pm

```diff
@@ -1,202 +1,108 @@
 package DDGC::DB::Result::User::Blog;
+
 # ABSTRACT: Result class of blog posts
 
-use Moose;
-use MooseX::NonMoose;
-extends 'DDGC::DB::Base::Result';
+use Moo;
```

Comment:
This looks largely cosmetic. Can you expound on what this gets us? It looks like it's removing a level of indirection from what the base class really should be?